tor_core/__init__.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import sys
+import time
+
+import prawcore
+
+from tor_core.config import config
+from tor_core.helpers import explode_gracefully
+
+__version__ = '3.0.0'
+
+
+default_exceptions = (prawcore.exceptions.RequestException)
+
+def run_until_dead(func, bot_name=None, exceptions=default_exceptions):
```

Comment:
The only reason it's in here is so that it's one import level higher than anything else; though now that I've slept on it, putting it in helpers might make more sense.